### PR TITLE
Sprint12 05 a47353 app tomcat mod jk vhost

### PR DESCRIPTION
**Pull request by Neve**
fixes:
https://wush.net/trac/rightscale/ticket/4678 (app_tomcat put mod_jk vhost directly into {apache}/sites-available)

also misc:
removed direct use of node[:app] variables, 
add "app_add_listen_port" instead of local template

### CODE REVIEW

File: cookbooks/app_tomcat/templates/default/apache_mod_jk_vhost.erb [hunk 1]
```diff
@@ -72,8 +72,8 @@
      # <CUSTOM RULES BEFORE FORWARDING END>
 
      # Setup the logs in the appropriate directory
-     CustomLog <%= @apache_log_dir %>/access_log combined
-     ErrorLog  <%= @apache_log_dir %>/error_log
+     CustomLog <%=@params[:apache_log_dir]%>/<%=@params[:server_name]%>-access_log combined
```
Comment (ekargard) [inline cookbooks/app_tomcat/templates/default/apache_mod_jk_vhost.erb:75]: Was a ticket created for adding the server name to the log file?

Comment (Neve) [inline cookbooks/app_tomcat/templates/default/apache_mod_jk_vhost.erb:75]: nope, it was just quick enhancement.

Comment (Neve) [inline cookbooks/app_tomcat/templates/default/apache_mod_jk_vhost.erb:75]: copy-pasted it from passenger-rails vhost file :)

**Comment by ekargard** [comment]
Looks good merge

**Pull request merged**